include/wx/dcsvg.h

```diff
@@ -66,7 +66,8 @@ class WXDLLIMPEXP_CORE wxSVGFileDCImpl : public wxDCImpl
 {
 public:
     wxSVGFileDCImpl( wxSVGFileDC *owner, const wxString &filename,
-                     int width=320, int height=240, double dpi=72.0 );
+                     int width=320, int height=240, double dpi=72.0,
```

Comment:
Should try to keep the code style consistent by using spaces around the assignment operator:
```c++
int width = 320, int height = 240, double dpi = 72.0